data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -93,17 +128,19 @@
                                 description= _ "This massive blade was created centuries ago by long-forgotten elvish forgemasters, who imbued the bluish steel with an inner magical fire. Tongues of flame dance on the surface, giving the metal a flawless mirrored finish."
                                 cannot_use_message= _ "Only the leader of an army can wield this sword!"
                                 [filter]
-                                    type=Fighter,Commander,Lord,Princess,Battle Princess,Elvish Captain,Elvish Hero,Elvish Marshal,Elvish Champion,Paladin,Elvish Lord,Elvish High Lord
+                                    type=Fighter,Commander,Lord,Princess,Battle Princess,Elvish Captain,Elvish Hero,Elvish Marshal,Elvish Champion,Elvish Lord,Elvish High Lord
```

Comment:
Actually, I think Paladin seems like an entirely reasonable choice. From this point on, you'll encounter almost exclusively arcane-resistant enemies, so the normal sword mostly sucks. Getting magical, +25% damage and switching the damage type from a liability to a neutral (and occasionally good) one doesn't seem bad at all.